ui/pages/routes/routes.component.js

```diff
@@ -369,11 +367,6 @@ export default class Routes extends Component {
           component={ConfirmTransaction}
         />
         <Authenticated path={SEND_ROUTE} component={SendPage} exact />
-        <Authenticated
-          path={`${TOKEN_DETAILS}/:address/`}
-          component={TokenDetailsPage}
-          exact
-        />
```

Comment:
I considered trying to maintain this route and point it to the main asset page.  But I think its unlikely anyone is keeping bookmarks to it.

Reply:
Looks like removing this redirects to the TokenList page. That makes sense to me 👍